### PR TITLE
Remove validator parts

### LINKED ISSUE #11: Separate schema and validation library
__What should we do?__

We should split up the general schema and the validation library to be able to release both independently.

[Excalidraw Sketch](https://excalidraw.com/#json=6325902669512704,10KZljquAIn4U1TET2AV4A)

image

**naming of the packages**
@bpmn-io/json-schema-validator
camunda-element-template-json-schema

__Why should we do it?__

Cf. https://github.com/bpmn-io/element-templates-json-schema/issues/5#issuecomment-718816896

Child of #5 
Related to https://github.com/bpmn-io/internal-docs/issues/199

### PR DESCRIPTION
**Pull request by pinussilvestrus**
__Which issue does this PR address?__

* Moves the validator library part to [`bpmn-io/json-schema-validator`](https://github.com/bpmn-io/element-templates-validator/pull/1)
* Changes the test setup
* Adds an usage example on top of [`bpmn-io/json-schema-validator`](https://github.com/bpmn-io/element-templates-validator)

Closes #11 

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/.github/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
We should transform this project to `@camunda` once we clarify where to publish it.

Also, I tried out https://github.com/ajv-validator/ajv-istanbul to keep the coverage functionality but wasn't able to hook it into. :(

**Comment by pinussilvestrus** [comment]
Before merging this I want to incorporate a usage example based on the `json-schema-validator` library, cf.  https://github.com/bpmn-io/element-templates-validator/pull/1

**Comment by pinussilvestrus** [comment]
Update: I added a usage example as we considered in the Kickoff inside the project, cf. https://github.com/bpmn-io/element-templates-json-schema/pull/12/commits/547e66f012a6d8ac8038fab2123705c12e661ac3

This part should be ready to review then. What's missing from closing the epic (cf. #5), but should be handled separately
* Transform this project to @camunda GitHub organization
* Add additional combination validations, cf. #9 
* Maybe: Push the schema to somewhere meaningful so the `$schema` attribute does not have to rely on `unpkg`, cf. #6

File: README.md [hunk 1]
```diff
@@ -4,88 +4,22 @@
...
-  validTemplates,
-  erroredTemplates
-} = validateTemplates(templates);
-
-if (erroredTemplates.length) {
-  console.error('Invalid element templates detected:', erroredTemplates);
-}
-
-```
+Additionally, it's possible to use the schema on top of existing validation libraries. Follow the [example] for further instructions.
```
Comment (nikku) [inline README.md:22]: The only thing I see missing is the link to the example. Other than that :rocket:

File: example/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+$ npm install --save @bpmn-io/json-schema-validator
+```
+
+## Usage
+
+Import the schema and the validation library into your application.
+
+```js
+import {
+ Validator
```
Comment (nikku) [inline example/README.md:20]: Missing space? :see_no_evil:

File: example/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+} from '@bpmn-io/json-schema-validator';
+
+import schema from 'camunda-element-templates-json-schema/resources/schema.json';
+```
+
+Wrap the schema into the validator and run the validation with an example template.
+
+```js
+const validator = new Validator({
+ schema: schema
```
Comment (nikku) [inline example/README.md:30]: Same here.

File: README.md (hunk 1 above)
Comment (pinussilvestrus) [reply README.md:22]: 🙈

**Review by nikku — APPROVED** [review]
:clap: Looks good to me.

**Pull request merged**